playground.js

```diff
@@ -0,0 +1,12 @@
+import {$} from './index.js';
+
+let $$ = $({stdio: 'inherit'});
+
+await $$({shell: true})`ls *.ts`;
+
+await $$`echo "$TEST"`;
+
+$$ = $$({shell: true, env: {TEST: 'test'}});
+
+// eslint-disable-next-line no-unused-expressions
+$$.sync`echo "$TEST"`;
```

Comment:
I am wondering whether you intended to commit this file, or whether it was meant for debugging instead?

Reply:
Thanks for flagging. Yeah, this file was temporarily committed for debugging and exploring the different options binding APIs we discussed above. Removed: https://github.com/sindresorhus/execa/pull/510/commits/3f6de3d730bbbe3ce54a426f6bef8213d74cfea3